.gitignore

```diff
@@ -1,5 +1,15 @@
-.DS_Store
+# compiled binaries
+bin
+
+# direnv
+.envrc.local
+
+# terraform
 .terraform
+.terraform-version
 terraform.tfstate
 terraform.tfstate.backup
-terraform.tfstate.*.backup
\ No newline at end of file
+terraform.tfstate.*.backup
```

Comment:
it seems like you're removing and then adding the same line here?

Reply:
Adding a newline character to a file will make this appear as though it's being removed/added.